Core/Utilities/test/Test_Encode.py

```diff
@@ -0,0 +1,160 @@
+"""Test Encoding function of DIRAC
+Any library that would pretend replacing diset should pass this tests.
+
+"""
+
+
+from string import printable
+import sys
+
+from DIRAC.Core.Utilities.DEncode import encode as disetEncode, decode as disetDecode, g_dEncodeFunctions
+
+from hypothesis import given
```

Comment:
I don't know about this package, but anyway if it's in requirements.txt then it should also be in https://github.com/DIRACGrid/Externals/blob/master/FullServerPackages/dirac-make.py

Reply:
Ah ? Even if it is just for tests ?

Reply:
The linked file is for a "fullServerPackage" which is the most complete of the "Externals" installation. It just adds some testing packages that normally we don't need in the "Server" package. This is used in Jenkins only I think.

Reply:
Done in a PR